kairon/api/app/routers/user.py

```
@@ -244,7 +244,7 @@ async def get_model_testing_logs_accuracy(
 async def leave_bot(
     background_tasks: BackgroundTasks,
     bot: str = Path(description="bot id", examples=["613f63e87a1d435607c3c183"]),
-    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS)
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=AGENT_ACCESS)
```

Ruff

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)